post-processor/vsphere-template/step_create_folder.go

```diff
@@ -0,0 +1,72 @@
+package vsphere_template
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/packer/packer"
+	"github.com/mitchellh/multistep"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+)
+
+type StepCreateFolder struct {
+	Folder string
+}
+
+func (s *StepCreateFolder) Run(state multistep.StateBag) multistep.StepAction {
+	ui := state.Get("ui").(packer.Ui)
+	ctx := state.Get("context").(context.Context)
+	f := state.Get("finder").(*find.Finder)
+	d := state.Get("datacenter").(string)
```

Comment:
And `dc` here. In general try to avoid one character variable names, except fort the method type (`s`).